app/models/translation.rb

```diff
@@ -0,0 +1,5 @@
+class Translation < ActiveRecord::Base
+  attr_accessible :time_assigned
```

Comment:
You changed the field name in the model, but you didn't change it in the database. You'll have to roll back your migration, update the migration, and reapply it.